plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotTableProperties.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.session.PropertyMetadata;
+
+import java.util.List;
+
+import static io.trino.spi.session.PropertyMetadata.stringProperty;
+
+public class PinotTableProperties
+{
+    public static final String PINOT_COLUMN_NAME_PROPERTY = "pinot_column_name";
```

Comment:
Could you leave code comment why this property is required? It's much different from other connectors.

Reply:
Added a note. Do you recommend I do it differently? lmk if you have an example I can look at. Thanks!

Reply:
Could you push the change? I can't see the note.

Reply:
Pushed.